turbopack/crates/turbopack-core/src/chunk/availability_info.rs

```diff
@@ -17,40 +25,48 @@ use super::available_modules::{AvailableModules, AvailableModulesSet};
     Serialize,
     Deserialize,
 )]
-pub enum AvailabilityInfo {
-    /// Availability of modules is not tracked
-    Untracked,
```

Comment:
I guess there was no place we actually constructed an `AvailabilityInfo`?

Reply:
Yeah `Untracked` was dead code